lib/github_changelog_generator/parser.rb

```diff
@@ -185,6 +185,9 @@ def self.setup_parser(options)
         opts.on("--[no-]verbose", "Run verbosely. Default is true") do |v|
           options[:verbose] = v
         end
+        opts.on("--[no-]stdout", "Print changelog to stdout. When this is enabled, --output is ignored. Default is false") do |stdout|
+          options[:stdout] = stdout
+        end
```

Comment:
Another way to solve this would be to give the value `-` a special meaning in the `--output` option.

Suggestion for documentation change in `output`:

```ruby
opts.on("-o", "--output [PATH]", "Output file path. To print to STDOUT instead, use - as path. Default is CHANGELOG.md") do |last|
```

The logic in the `ChangelogGenerator` class would be about the same, but we wouldn't have to introduce and document a new option.

Reply:
I see! I added `--output -` instead of `--stdout`